.github/workflows/pr-license-check.yaml

```diff
@@ -18,11 +18,14 @@ jobs:
       - name: Check License
         run: |
           set -ex
+          python3 -m pip install --upgrade pip
           python3 -m pip install liccheck
+
           license_files=$(find -type f -name "requirements.txt")
```

Comment:
I don't find "requirements.txt" file in folder.
1) Do we still need this .txt file?
2) Or do we need to create a such file?

Reply:
It is under the folder cnap.